frontend/src/app/components/filters-form/filters-form.component.ts

```diff
@@ -1,24 +1,38 @@
-import { Component, inject, OnInit } from "@angular/core";
+import { Component, inject, OnInit ,EventEmitter, Output} from "@angular/core";
 import { TaskStatus } from "../../models/task-status";
 import { AppStore } from "../../store/app.store";
 import { AppState } from "../../models/app-state";
+import { FormControl } from "@angular/forms";
 
 @Component({
   selector: 'app-filters-form',
-  templateUrl: 'filters-form.component.html'
+  templateUrl: 'filters-form.component.html',
+  styleUrls: ['filters-form.component.scss']
 })
 export class FiltersFormComponent implements OnInit {
   protected TaskStatus = TaskStatus;
   protected appStore: AppStore = inject(AppStore);
   protected currentStatus: TaskStatus;
+  public formSearchString: FormControl = new FormControl('',);
+  @Output() taskFormVisible = new EventEmitter<boolean>();
+  //public taskFormVisible: boolean = false;
 
   ngOnInit(): void {
     this.appStore.state$.subscribe((state: AppState) => {
       this.currentStatus = state.filters.status;
     });
   }
 
-  applyFilters(status?: TaskStatus):void {
+  applyFilters(status?: TaskStatus): void {
```

Comment:
Rename `applyFilters` to `setStatusFilter` or `applyStatusFilter` so the name better reflects what it does.

Reply:
Done